cmd/context.go

```diff
@@ -156,9 +148,15 @@ var getContextCmd = &cobra.Command{
 			if context.ProviderID != "" {
 				t.AppendRow(table.Row{"Provider ID", context.ProviderID})
 			}
+			isAuth := "no"
+			if IsAuthorised() {
+				isAuth = fmt.Sprintf("yes, until %s", context.AccessTokenExpiry.Format(time.RFC822))
```

Comment:
May be confusing to a user (see other expiry comments)

Reply:
Yes, got it. I want to be a bit more descriptive here as this is also the way to get the access token to be used potentially somewhere else, so you may want to know. Changed it to "yes, refreshed after %s"